### PR TITLE
release 1.2.0

### PR DESCRIPTION
**Pull request by oliver-sanders**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
@conda-forge-admin, please rerender

**Comment by oliver-sanders** [comment]
Please check dependency changes match expectations.

**Comment by wxtim** [comment]
@oliver-sanders  rx has no lower-pin in the setup.cfg (`rx<2`.

All other deps match

**Comment by oliver-sanders** [comment]
(waiting for cylc-flow 8.1.0 to become available before re-running tests)

**Comment by oliver-sanders** [comment]
Pinned Python to `>=3.7,<3.10` as our dependencies are inflicting the upper pin.

**Comment by oliver-sanders** [comment]
@conda-forge-admin, please rerender

**Comment by oliver-sanders** [comment]
(in case fiddling Python version range matters)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,17 +32,17 @@ outputs:
...
-        - python >=3.7
-        - cylc-flow-base >=8.0,<8.1
+        - python >=3.7,<3.10
+        - cylc-flow-base >=8.1,<8.2
         - tornado >=6.1
         - graphene-tornado >=2.6.0,<2.7
         - graphql-ws ==0.4.4
         - jupyter_server >=1.10.2,<2.0
-        - traitlets >=5.0
+        - traitlets >=5.2.1
```
Comment (MetRonnie) [inline recipe/meta.yaml:45]: Good spot, we missed this in the last release

**Review by MetRonnie — APPROVED** [review]

**Pull request merged**